agent/command/attach_artifacts_test.go

```diff
@@ -102,6 +113,17 @@ func (s *ArtifactsSuite) TestArtifactErrorsIfDoesNotExist() {
 	s.cmd.Files = []string{"foo"}
 	s.Error(s.cmd.Execute(s.ctx, s.comm, s.logger, s.conf))
 	s.Len(s.cmd.Files, 0)
+	s.Len(s.mock.AttachedFiles, 0)
```

Comment:
I see that this is tested here, but I'd still strongly advocate to test this manually in staging as well, since historically file changes have been prone to bugs that were hard to discover from unit tests.